nrf5340-app-hal/log.log

```diff
@@ -0,0 +1,3527 @@
+[2021-Dec-03 05:29:48] [ info] --------------------------------------------------------------------------------
```

Comment:
Looks like this was checked in by accident

Reply:
Oh yup. Just removed it